packages/frontapp/src/app/components/ListBooks/index.tsx

```diff
@@ -52,6 +52,7 @@ const ListItems: FC<IProps> = ({ items }) => {
         items.map((item) => (
           <BookCard
             key={item?.id}
+            item_id={item?.id}
```

Comment:
use item.material.id here, because you need the id of the material, not its status